packages/block-editor/src/components/spacing-sizes-control/utils.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Checks is given value is a spacing preset.
+ *
+ * @param {string} value Value to check
+ *
+ * @return {boolean} Return true if value is string in format var:preset|spacing|.
+ */
+export function isValueSpacingPreset( value ) {
+	if ( ! value?.includes ) {
+		return false;
+	}
+	return value.includes( 'var:preset|spacing|' );
+}
+
+/**
+ * Converts a spacing preset into a custom value.
+ *
+ * @param {string}  value Value to convert
+ * @param {Array}   spacingSizes Array of the current spacing preset objects
```

Comment:
```suggestion
 * @param {string} value        Value to convert
 * @param {Array}  spacingSizes Array of the current spacing preset objects
```

I'm getting a couple of ESLint warning regarding misaligned JSDoc comments.

Reply:
hmm, ESLint seems to have packed a sad in my VS Code - fixed